tests/report_downstream_test_diffs.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import json
+import os
+import re
+import subprocess
+import tempfile
+from collections import namedtuple
+
+TestCase = namedtuple("TestCase", ["api", "file", "test"])
+
+
+def pytest_test_cases():
+    p = subprocess.run(
+        ["pytest", "--browser", "chromium", "--collect-only", "-q"],
+        stdout=subprocess.PIPE,
+    )
+    regex = (
+        r"tests/(?P<api>a?sync)/test_(?P<file>.*)\.py::test_(?P<test>.*)\[chromium\]"
+    )
+    matches = re.finditer(regex, p.stdout.decode(), re.MULTILINE)
+    for match in matches:
+        yield TestCase(
+            match.group("api"), match.group("file"), match.group("test"),
+        )
+
+
+def jest_test_cases(playwright_js_path):
+    env = os.environ.copy()
+    env["REPORT_ONLY"] = "true"
+    with tempfile.NamedTemporaryFile() as results:
+        subprocess.run(
+            ["npx", "jest", "--json", "--outputFile", results.name],
```

Comment:
@pavelfeldman Is there a report-only option for the new test runner? I see jest runner is now removed.

Reply:
I'll bring it back shortly.

Reply:
Thanks! Updated to use the new test runner 😄 🎉